openstates/nc/bills.py

```diff
@@ -1,10 +1,15 @@
 import datetime as dt
 import lxml.html
+import re
 from pupa.scrape import Bill, Scraper, VoteEvent
 import pytz
+from collections import defaultdict
+
 
 eastern = pytz.timezone("US/Eastern")
 
+archived_votes = defaultdict(lambda: defaultdict(list))
```

Comment:
can you move this to self.archived_votes

Reply:
Moved!